.github/scripts/check-release.sh

```diff
@@ -9,5 +9,12 @@ if [ "$current_tag" != "$file_tag" ]; then
   exit 1
 fi
 
+package_version_ts=$(grep "PACKAGE_VERSION =" src/package-version.ts | cut -d "=" -f 2- | tr -d " " | tr -d "'")
+if [ "$current_tag" != "$package_version_ts" ]; then
+  echo "Error: the current tag does not match the version in src/package-version.ts."
+  echo "$current_tag vs $package_version_ts"
+  exit 1
+fi
```

Comment:
What we do in the Meilisearch repo. Don't know if it's exactly the same usecase, but it can maybe help: https://github.com/meilisearch/meilisearch/blob/release-v0.28.0/.github/scripts/check-release.sh